uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -76,7 +95,7 @@ def connection_made(self, transport):
         self.client = get_remote_addr(transport)
         self.scheme = "wss" if is_ssl(transport) else "ws"
 
-        if self.logger.level <= TRACE_LOG_LEVEL:
+        if self.logger.isEnabledFor(TRACE_LOG_LEVEL):
```

Comment:
why change this ?

Reply:
details here https://github.com/aaugustin/websockets/issues/1021

Reply:
so if I follow correctly the logger inside this module is not the standard python logger anymore but a LoggerAdapter that we are kind of plugging from websockets ?

Reply:
it depends, but logger.level is private undocumented and you're supposed to use isEnabledFor